core/funds.py

```diff
@@ -0,0 +1,16 @@
+from connexion import NoContent
+from core.data_operations import fund_data
+
+# from flask import request
+
+
+def get_funds():
+    return fund_data.FUNDS_DUMMY_DAO.get_all()
```

Comment:
🔧 Let's add a nice status code to this :) We should also add a "fail" outcome to this - IE a 404.

Reply:
💯 done!